<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JunSuCpp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_workspace">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="windowTitle">
    <string>工作空间</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_layerlist">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255, 255);</string>
   </property>
   <property name="windowTitle">
    <string>图层列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_properties">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="windowTitle">
    <string>属性信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_ouput">
   <property name="minimumSize">
    <size>
     <width>87</width>
     <height>87</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>输出信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开工作空间或数据源文件</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存工作空间</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为工作空间</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
